Description: Only list Python3 version currently available in Debian testing/unstable (targetting bookworm).
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py32,py33,py34,py35,py36
+envlist = py39,py310
 
 [testenv]
 commands = python -m unittest discover -vv click_package.tests
